src/mame/trs/coco12.cpp

```diff
@@ -562,6 +689,37 @@ void coco12_state::cocoeh(machine_config &config)
 	m_ram->set_default_size("64K");
 }
 
+void deluxecoco_state::deluxecoco(machine_config &config)
+{
+	coco12_state::coco2b(config);
+
+	m_ram->set_default_size("64K");
+
+	// Asynchronous Communications Interface Adapter
+	mos6551_device &acia(MOS6551(config, MOSACIA_TAG, 0));
+	acia.set_xtal(1.8432_MHz_XTAL);
+	acia.irq_handler().set(m_irqs, FUNC(input_merger_device::in_w<2>));
+	acia.txd_handler().set(ACIA_TAG, FUNC(rs232_port_device::write_txd));
+	acia.rts_handler().set(ACIA_TAG, FUNC(rs232_port_device::write_rts));
+	acia.dtr_handler().set(ACIA_TAG, FUNC(rs232_port_device::write_dtr));
+
+	rs232_port_device &rs232(RS232_PORT(config, ACIA_TAG, default_rs232_devices, nullptr));
+	rs232.set_option_device_input_defaults("null_modem", DEVICE_INPUT_DEFAULTS_NAME(acia));
+	rs232.set_option_device_input_defaults("pty", DEVICE_INPUT_DEFAULTS_NAME(acia));
```

Comment:
Should you set the defaults for the HLE text terminal as well?

Reply:
At first I was confused that this didn't set the default for all serial options. But you ware right, Terminal is a real option as well.